packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
another version nit

Reply:
As per external convo, have decided to stick with 0.8.15 for now and will make a more concerted push to move to 0.8.25 separately.